packages/devops/scripts/ci/updateCiEnvVars.sh

```diff
@@ -1,6 +1,41 @@
 #!/bin/bash
+
+# Set default branch as current branch
+COMPARE_BRANCH=""
+
+# Getting command line arguments, taken from https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case $key in
+    --compare)
+        COMPARE_BRANCH="$2"
+        shift # past argument
+        shift # past value
+        ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
 rm -f .env
-for PACKAGE in "meditrak-server" "admin-panel" "web-frontend" "web-config-server" "devops" "database"; do
+rm -f old.env
+echo "Checking out previous ci-env-vars"
+git reset -q HEAD ci-env-vars.encrypted
+git checkout $COMPARE_BRANCH -- ci-env-vars.encrypted
+git reset -q HEAD ci-env-vars.encrypted
+echo "Decryting old ci-env-vars"
+jet decrypt ci-env-vars.encrypted old.env
+echo "Creating new ci-env-vars"
+for PACKAGE in "meditrak-server" "admin-panel" "web-frontend" "web-config-server" "devops" "data-api" "database"; do
```

Comment:
Silly question but any reasons that we only want to update CI env vars for these packages?

Reply:
So these are the only packages that require an `.env` file to run their tests (except for devops which I think uses an env var as part of the codeship script itself).

I suppose we could change it to use all packages (and update the script to handle when that .env file isn't present) however then we have a little bit more complexity ensuring there are no name clashes between the .env files (something that we should avoid anyway in fairness)